src/sentry/models/integration.py

```diff
@@ -13,10 +13,26 @@
 from sentry.signals import integration_added
 
 
+class PagerDutyService(Model):
+    __core__ = False
+
+    organization_integration = FlexibleForeignKey("sentry.OrganizationIntegration")
+    integration_key = models.CharField(max_length=255)
+    service_id = models.CharField(max_length=255)
+    service_name = models.CharField(max_length=255)
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_pagerdutyservice"
+        unique_together = (("service_id", "integration_key"),)
+
+
 class PagerDutyServiceProject(Model):
     __core__ = False
 
     project = FlexibleForeignKey("sentry.Project", db_index=False, db_constraint=False)
+    pagerduty_service = FlexibleForeignKey("sentry.PagerDutyService")
+    # TODO(meredith): rest of these columns need to be removed
```

Comment:
@MeredithAnya why can't we remove them now?

Reply:
We need to make them nullable before we can remove them. We could do that in this PR though.

Steps for removing columns are (in 3 prs that are deployed separately):
 - Make the columns you want to remove nullable if they aren't.
 - Remove the columns from the model. Generate the migration, but remove the code that actually deletes from the database (save it for step 3)
 - Generate another migration and put the database deletions in that.